pkg/registry/core/service/allocator/bitmap_test.go

```diff
@@ -155,3 +155,37 @@ func TestContiguousAllocation(t *testing.T) {
 		t.Errorf("unexpected success")
 	}
 }
+
+func allocateAllBitmap(max int, m Interface) {
+	for i := 0; i < max; i++ {
+		m.AllocateNext()
+	}
+}
+
+func benchmarkContiguousAllocationBitmap(b *testing.B, max int) {
+	m := NewContiguousAllocationMap(max, "test")
```

Comment:
As far as I can tell NOBODY uses this - should we just remove contiguous entirely?

Reply:
it's about 12 lines of code and it can be used as an example or to compare against it, like in this benchmark.
I will keep it for now.

Reply:
just remove the commit with the bitmap benchmark, is unrelated to the PR, and I'm not sure the benchmark is correct or at least meaningful in a way it can be used as an example